.github/workflows/manual-upgrade-testnet-l2.yml

```diff
@@ -184,3 +190,18 @@ jobs:
         run: |
           ./.github/workflows/runner-scripts/wait-node-healthy.sh --host=obscuronode-0-${{needs.build.outputs.RESOURCE_TESTNET_NAME}}-${{needs.build.outputs.VM_BUILD_NUMBER}}.uksouth.cloudapp.azure.com
           ./.github/workflows/runner-scripts/wait-node-healthy.sh --host=obscuronode-1-${{needs.build.outputs.RESOURCE_TESTNET_NAME}}-${{needs.build.outputs.VM_BUILD_NUMBER}}.uksouth.cloudapp.azure.com --timeout=60
+
+  deploy-faucet:
+    runs-on: ubuntu-latest
+    needs:
+      - check-obscuro-is-healthy
+    steps:
+      - name: 'Trigger Faucet deployment for testnet'
+        if: ${{ github.event.inputs.testnet_type  == 'testnet' }}
```

Comment:
Bit of a shame to have use two steps with 'if' statements if there's just one minor difference between the two (the event type is dev_testnet_deployed), I think something like this would work instead of the if:

```
# event_type field requires the testnet type but with an underscore
eventtype="${github.event.inputs.testnet_type/-/_}_deployed"

curl -XPOST -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" -H "Accept:application/vnd.github" -H "Content-Type:application/json" https://api.github.com/repos/obscuronet/faucet/dispatches --data '{ "event_type": "${eventtype}", "client_payload": { "ref": "${{ github.ref }}", "env": "${{ github.event.inputs.testnet_type }}", "event": "${{ github.event_name }}" }'
```
Tested the replace locally:
<img width="283" alt="image" src="https://github.com/obscuronet/go-obscuro/assets/98158711/e7a10a6d-8999-474e-8849-372c9e35d1d2">

Reply:
You can use the same trick to avoid the branching on the health check step above